routes/index.js

```diff
@@ -1,7 +1,6 @@
 const express = require('express');
```

Comment:
Let's remove this route - it's no longer meaningful in the context of this API (we will need to remove the corresponding `require` and `app.use` in `app.js`)